src/main/java/command/Command.java

```diff
@@ -0,0 +1,40 @@
+package command;
+
+import task.TaskList;
+import ui.UserInterface;
+import duke.Storage;
+
+/**
+ * Abstract Command class that specifies the skeleton of all commands.
+ * Commands are actions that the user can request.
+ */
+public abstract class Command {
+    //Only TRUE if commands = bye which indicates the end of session.
```

Comment:
I really like your little comments throughout the code. I think it will be really helpful to people new to the project/code!